tests/robot-tests/tests/admin_and_public/bau/subject_reordering.robot

```diff
@@ -182,6 +182,17 @@ Add data guidance for all subjects
 Publish release
     user approves original release for immediate publication
 
+Check subjects can no longer be re-ordered after release has been published
+    user clicks link    Data and files
+    user waits until page contains element    id:uploadedDataFiles
+
+    user checks accordion is in position    One    1    id:uploadedDataFiles
+    user checks accordion is in position    Two    2    id:uploadedDataFiles
+    user checks accordion is in position    Three    3    id:uploadedDataFiles
+    user checks accordion is in position    Four    4    id:uploadedDataFiles
+
+    user checks element is not visible    //*[button[text()="Reorder"]]    %{WAIT_SMALL}
```

Comment:
This selector would never fail as it's matching on exact text and the button has visually hidden text that is breaking up the element.

Would probably be best to use a test id instead